subsys/net/ip/net_shell.c

```diff
@@ -59,7 +59,6 @@ LOG_MODULE_REGISTER(net_shell, LOG_LEVEL_DBG);
 #endif
 
 #if defined(CONFIG_NET_L2_PPP)
-#include <net/ppp.h>
```

Comment:
Hmm, wondering if we should remove the one above, and leave this one with ifdef protection.  @jukkar ?

Reply:
Yes, better to remove the one above which is without ifdef protection.